ccp/index_reader.py

```diff
@@ -366,14 +367,29 @@ def __init__(self, index, registry_url, namespace,
             registry_url, namespace, from_address, smtp_server)
         self.infra_projects = ["seed-job"]
 
-    def find_stale_jobs(self, oc_projects, index_projects):
+    def find_stale_jobs(self,
+                        oc_projects, index_projects,
+                        exception_jobs=["ci-job"]):
         """
```

Comment:
let's add this to calling function. Also adding this list to some constant makes more sense. we need to update the list to '[ci-success-job, ci-failure-job]`